RELEASING.md

```diff
@@ -28,7 +28,7 @@
   - [Requesting a Release](#requesting-a-release)
   - [Security Reviews](#security-reviews)
 
-## Overview
+ Overview
```

Comment:
I'm OK with this, but could you please update the TOC section of the doc too?

Reply:
Thas werid..  I didn't touch, I thought line 31...   Not sure why the `##` went away..   And I thought that my change actually MAKES the TOC work!  sigh.